docs/SETTINGS.rst

```diff
@@ -87,20 +87,20 @@ Consul
 ------
 Most of the parameters are optional, but you have to specify one of the **host** or **url**
 
--  **host**: the host:port for the Consul endpoint, in format: http(s)://host:port
--  **url**: url for the Consul endpoint
--  **port**: (optional) Consul port
--  **scheme**: (optional) **http** or **https**, defaults to **http**
--  **token**: (optional) ACL token
--  **verify**: (optional) whether to verify the SSL certificate for HTTPS requests
+-  **host**: the host:port for the Consul endpoint.
+-  **url**: url for the Consul endpoint.
+-  **port**: (optional) Consul port.
```

Comment:
I think for consistency with https://github.com/zalando/patroni/blob/7176a4155a61f46c6eb44c071c3e483eda09093f/docs/ENVIRONMENT.rst#consul it makes sense to move the `, in format: http(s)://host:port` to the line with `url`.

Since you already on it lets also replace `Consul` with `Consul local agent` on the lines with `host`, `port`, and `url` in this file and in the ENVIRONMENT.rst

Reply:
Ok, done, but does it have to be a local agent?  If so, why is host a parameter rather than always using localhost?  I assumed it could utilize a different host.  Also, isn't url duplication of schema, host, and port?

Reply:
> does it have to be a local agent?

I think that in most cases yes.

> If so, why is host a parameter rather than always using localhost?

What if agents listen on a host different from localhost for some reason?

> I assumed it could utilize a different host.

With an agent running on a different host there is way more uncertainty. What if the different host is down?

> Also, isn't url duplication of schema, host, and port?

Yes, it duplicates them, but someone might find the url more convenient to use, especially when using environment variables to configure Patroni. It is enough to define just one variable instead of three.